internal/dev/server.go

```
@@ -598,6 +594,45 @@ func (s *Server) monitor() {
 	}
 }
 
+var (
+	logoColor    = lipgloss.AdaptiveColor{Light: "#11c7b9", Dark: "#00FFFF"}
+	labelColor   = lipgloss.AdaptiveColor{Light: "#999999", Dark: "#FFFFFF"}
+	runningColor = lipgloss.AdaptiveColor{Light: "#00FF00", Dark: "#009900"}
```

golangci-lint (1.64.8)

var `runningColor` is unused

(unused)